community/contributing/managing-deprecations.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: single
+title: "Managing Deprecations"
+sidebar:
+  nav: community
+---
+
+Deprecating old functionality is key to making Spinnaker easier to maintain, but it needs to be done in a way that is as non-disruptive as possible.
+
+## Before you begin
+
+The best way to propose a deprecation is to first [open an
+issue](https://github.com/spinnaker/spinnaker/issues/new), ping the [#dev channel](https://spinnakerteam.slack.com/messages/C0DPVDMQE/) in slack and discuss your deprecation before taking action.
+
+Often times, you will need the following information:
+
+1. An overview of what the functionality does, or intended to do.
+1. A description of why this functionality should be deprecated, and eventually, removed.
+1. A description of the replacement functionality, if any, or information about why this functionality will not need a replacement.
+1. The scope of impact on the codebase: What services will be affected?
+1. How will users migrate off this functionality?[^1]
```

Comment:
apparently GitHub doesn't support footnotes

Reply:
Github flavored markdown does not, no. I did confirm that the markdown engine that our jekyll site uses does, though.